mkdocs_documentation/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove the vscode folder, also how many docs did you add?

Reply:
I have removed `mkdocs_documentation` folder. I added `jenkins_gitlab_integration` document.